k8s/README.md

```diff
@@ -0,0 +1,65 @@
+# Running *warp* on kubernetes
+This document describes with simple examples on how to automate running *warp* on kubernetes.
+
+## Create *warp* client listeners
+
+Create *warp* client listeners to run distributed *warp* benchmark, here we will run them as stateful sets across client nodes.
+
+```
+~ kubectl create -f warp.yaml
+```
+
+```
+~ kubectl get pods -l app=warp
+NAME     READY   STATUS    RESTARTS   AGE
+warp-0   1/1     Running   0          6m53s
+warp-1   1/1     Running   0          6m57s
+warp-2   1/1     Running   0          7m8s
+warp-3   1/1     Running   0          7m17s
+```
+
+Now prepare your *warp-job.yaml* (we have included a sample please edit for your needs) to benchmark your MinIO cluster
+```
+~ kubectl create -f warp-job.yaml
+```
+
+```
+~ kubectl get pods -l job-name=warp-job
+NAME             READY   STATUS      RESTARTS   AGE
+warp-job-6xt5k   0/1     Completed   0          8m53s
+```
+
+To obtain the console output look at the job logs
```

Comment:
Is it possible to get the file with the run statistics?

Reply:
Yes I need to add local persistence to get the output, but I think in such situations it is better to dump this output back to object storage?